doc/source/contribute/developer.rst

```diff
@@ -229,12 +229,55 @@ To modify the SCSS files, follow these steps:
 
 1. Navigate to the ``src/ansys_sphinx_theme/assets/styles/`` directory.
 2. Edit the relevant SCSS files as needed.
-3. Build the SCSS files using the following command:
+3. Build the library using the following command along with the documentation dependencies:
 
    .. code-block:: bash
 
       python -m pip install -e '.[doc]'
-      stb serve doc/source
+
+   This command installs the library in editable mode and builds the SCSS files.
+
+.. important::
+
+   The built SCSS files are stored in the
+   ``src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/styles/`` directory.
+   These files are regenerated during each build process, so avoid editing them directly.
+
+1. Build the documentation and serve it locally using any of the following commands:
+
+   .. tab-set::
+
+        .. tab-item:: using Tox (recommended)
+
+            **Linux/macOS/UNIX/Windows**
+
+            .. code-block:: bash
+
+                python -m tox -e doc-serve
```

Comment:
Cannot you do ``tox -e doc-server`` also (without the ``python -m`` part) ??

Reply:
It is possible to use both:

- Ensure Tox is the one of the active Python environment: `python -m tox -e doc-server`
- Use the first Tox in the `PATH`: `tox -e doc-server`